packages/tasit-sdk/src/Decentraland.forkedRopsten.test.js

```diff
@@ -0,0 +1,250 @@
+  addressesAreEqual,
+  bigNumberify,
+  etherFaucet,
+  ropstenManaFaucet,
+  constants,
+  ProviderFactory,
+  DecentralandUtils,
+} from "./testHelpers/helpers";
+
+const { ONE, TEN } = constants;
+
+const ROPSTEN_NETWORK_ID = 3;
+
+describe("Decentraland tasit app test cases (ropsten)", () => {
+  let ownerWallet;
+  let ephemeralWallet;
+  let manaContract;
+  let landContract;
+  let estateContract;
+  let marketplaceContract;
+  let landForSale;
+  let estateForSale;
+  let snapshotId;
+  let provider;
+
+  before("", async () => {
+    ConfigLoader.setConfig(config);
+
+    provider = ProviderFactory.getProvider();
+
+    const network = await provider.getNetwork();
+    const { chainId } = network;
+    expect(chainId, "The target network isn't ropsten.").to.equal(
+      ROPSTEN_NETWORK_ID
+    );
+
+    manaContract = new Mana(MANA_ADDRESS);
+    landContract = new Land(LAND_ADDRESS);
+    estateContract = new Estate(ESTATE_ADDRESS);
+    marketplaceContract = new DecentralandMarketplace(MARKETPLACE_ADDRESS);
+
+    const decentralandUtils = new DecentralandUtils();
+    const { getOpenSellOrders } = decentralandUtils;
+
+    const fromBlock = 0;
+    const openSellOrders = await getOpenSellOrders(fromBlock);
+
+    // Note: The exact amount of land isn't predictable since we are forking from the latest block
+    expect(openSellOrders).to.not.be.empty;
+
+    // Pick two open sell orders: one for a parcel of land and one for an estate
+    for (let sellOrder of openSellOrders) {
+      const { values: order } = sellOrder;
+      const { nftAddress, expiresAt } = order;
+
+      const isLand = addressesAreEqual(nftAddress, LAND_ADDRESS);
+      const isEstate = addressesAreEqual(nftAddress, ESTATE_ADDRESS);
+      const expired = Number(expiresAt) < Date.now();
+
+      // All parcels of land and estates for sale are expired (block 5058416) - otherwise we would select one that isn't expired
+      if (isLand) landForSale = order;
+      if (isEstate) estateForSale = order;
+
+      if (landForSale && estateForSale) break;
+
+      if (!isLand && !isEstate)
+        expect(
+          false,
+          "All land for sale should be an NFT that is either a parcel of land or an estate"
+        ).to.equal(true);
+    }
+
+    expect(estateForSale).not.to.be.an("undefined");
+    expect(landForSale).not.to.be.an("undefined");
+  });
+
+  beforeEach(
+    "buyer approves marketplace contract to transfer tokens on their behalf",
+    async () => {
+      snapshotId = await createSnapshot(provider);
+
+      ({ ownerWallet, ephemeralWallet } = setupWallets());
+      expect(ownerWallet.address).to.have.lengthOf(42);
+      expect(ephemeralWallet.address).to.have.lengthOf(42);
+
+      await etherFaucet(provider, ownerWallet, ephemeralWallet.address, ONE);
+
+      const { priceInWei: landPrice } = landForSale;
+      const { priceInWei: estatePrice } = estateForSale;
+
+      const manaAmountToShopping = bigNumberify(landPrice).add(
```

Comment:
```suggestion
      const manaAmountForShopping = bigNumberify(landPrice).add(
```

Reply:
@marcelomorgado I think the ropsten test suite will fail until we change this in a couple other spots too. But since we only run local unit tests on CircleCI, it’s passing nonetheless

Reply:
You are right. I'll fix that on next PR.

Reply:
Done: https://github.com/tasitlabs/TasitSDK/pull/214

Reply:
Thanks! Resolving